### ISSUE TITLE
the PSNR-oriented model with high PSNR performance

### ISSUE THREAD

**Issue by Jiequannnnnnnnnn** [xinntao/ESRGAN#62]
Which model does the **RRDB_PSNR_x4.pth: the PSNR-oriented model with high PSNR performance.** correspond to in the paper?

**Comment by xinntao** [comment]
RRDB_PSNR_x4.pth

https://drive.google.com/open?id=1pJ_T-V1dpb1ewoEra1TGSWl5e6H7M4NN

**Comment by Jiequannnnnnnnnn** [comment]
Hi, Thanks for your reply!  I was able to find this model. What I asked was that: is this model introduced in the paper? 
I was wondering if this model is PNSR oriented one used in network interpretation. Just want to make sure.
Also, is the loss function the only difference between RRDB_PSNR_x4.pth and RRDB_ESRGAN_x4.pth?
Appreciate your help and contribution!

**Comment by xinntao** [comment]
1) as the pre-trained model for ESRGAN. 2) for network interpolation.

Yes, they are trained by different losses.

**Comment by suesucong** [comment]
> RRDB_PSNR_x4.pth
> https://drive.google.com/open?id=1pJ_T-V1dpb1ewoEra1TGSWl5e6H7M4NN

Hi, why not just put the pretrained models into the dir?
